### PR TITLE
(kubernetes-kustomize) Added AU automatic package

### PR DESCRIPTION
**Pull request by wkritzinger**
## Description
Added new automatic package for [kustomize](https://github.com/kubernetes-sigs/kustomize), more or less copying the package format of kubernetes-kompose.

## Motivation and Context
This is a "new" tool that's becoming popular with Kubernetes and there was no easy installation method yet.

## How Has this Been Tested?
Just locally as it's basically a copy of kubernetes-kompose.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Migrated package (a package has been migrated from another repository)

## Checklist:

- [x] My code follows the code style of this repository.
- [ ] My change requires a change to documentation (this usually means the notes in the description of a package).
- [x] I have updated the documentation accordingly (this usually means the notes in the description of a package).
- [x] All files are up to date with the latest [Contributing Guidelines](https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md)
- [ ] The added/modified package passed install/uninstall in the chocolatey test environment.
- [x] The changes only affect a single package (not including meta package).

### CODE REVIEW

File: automatic/kubernetes-kustomize/kubernetes-kustomize.nuspec [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    <title>Kustomize: Customize Kubernetes YAML definitions</title>
+    <authors>Kubernetes authors</authors>
+    <projectUrl>https://github.com/kubernetes-sigs/kustomize</projectUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/e4e529ccc804ac6fac259a658e395e38b42b24f3/icons/kubernetes-kompose.png</iconUrl>
+    <licenseUrl>https://github.com/kubernetes-sigs/kustomize/blob/master/LICENSE</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <projectSourceUrl>https://github.com/kubernetes-sigs/kustomize</projectSourceUrl>
+    <docsUrl>https://github.com/kubernetes-sigs/kustomize</docsUrl>
+    <bugTrackerUrl>https://github.com/kubernetes-sigs/kustomize/issues</bugTrackerUrl>
+    <tags>kubernetes kustomize yaml devops cli foss</tags>
```
Comment (AdmiringWorm) [inline automatic/kubernetes-kustomize/kubernetes-kustomize.nuspec]: please add the `cross-platform` tag as well

File: automatic/kubernetes-kustomize/kubernetes-kustomize.nuspec [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>kubernetes-kustomize</id>
```
Comment (majkinetor) [inline automatic/kubernetes-kustomize/kubernetes-kustomize.nuspec]: why prefix, just kustomize and just tag it with kubernetes

File: automatic/kubernetes-kustomize/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+# <img src="https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/master/icons/kubernetes-kustomize.png" width="48" height="48"/> [kubernetes-kustomize](https://chocolatey.org/packages/kubernetes-kustomize)
+
+## Kustomize
```
Comment (majkinetor) [inline automatic/kubernetes-kustomize/README.md]: This should be in "standard" Readme form

See https://github.com/chocolatey/chocolatey-coreteampackages/blob/master/CONTRIBUTING.md#123-informative-description

**Comment by AdmiringWorm** [comment]
@wkritzinger are you able to fix the requested changes?

I'm afraid we cannot accept this PR until the requested changes is fixed.

**Comment by wkritzinger** [comment]
Hey! Thanks for the detailed instructions on how to get it accepted. I'll
be able to fix it, but I haven't had a lot of free time lately. I'll make
time for it soon!

**Comment by majkinetor** [comment]
Closed due to no activity.

**Pull request closed without merging**